src/proxy-ui-api/src/test/java/org/niis/xroad/restapi/openapi/IdentifierValidationRestTemplateTest.java

```diff
@@ -184,4 +184,44 @@ private void assertUpdateServiceDescriptionValidationFailure(String restServiceC
         assertEquals("validation_failure", errors.get("code"));
     }
 
+    @Test
+    public void initialServerConf() {
+        assertInitialServerConfValidationError(HAS_COLON, "aa", "aa");
+        assertInitialServerConfValidationError(HAS_SEMICOLON, "aa", "aa");
+        assertInitialServerConfValidationError(HAS_PERCENT, "aa", "aa");
+        assertInitialServerConfValidationError(HAS_NON_NORMALIZED, "aa", "aa");
+        assertInitialServerConfValidationError(HAS_BACKSLASH, "aa", "aa");
+        assertInitialServerConfValidationError("aa", HAS_COLON, "aa");
+        assertInitialServerConfValidationError("aa", HAS_SEMICOLON, "aa");
+        assertInitialServerConfValidationError("aa", HAS_PERCENT, "aa");
+        assertInitialServerConfValidationError("aa", HAS_NON_NORMALIZED, "aa");
```

Comment:
Nicely tested validation :+1:

Reply:
Also seemed to (randomly, not always) exhaust the default 20 requests per second IP throttling rate. Increased the rate to 60/second for tests.

Reply:
Oh, did not realize that could become an issue. Good change then. Hopefully that wont randomly fail even with the increased limit. Might also think about removing throttle for the tests altogether.

Reply:
throttle filter was disabled altogether from tests by @carohauta